src/plugins/intel_cpu/src/compiled_model.h

```diff
@@ -93,6 +93,7 @@ class CompiledModel : public ov::ICompiledModel {
     std::vector<std::shared_ptr<CompiledModel>> m_sub_compiled_models;
     std::shared_ptr<SubMemoryManager> m_sub_memory_manager = nullptr;
     bool m_has_sub_compiled_models = false;
+    bool m_is_single_thread = false;
```

Comment:
This flag emphasizes not only single thread scenario but also the single stream. However, the name of the flag omits the second part, which in fact is more important. Suggest renaming the flag to "m_optimized_single_stream".

Reply:
updated